CONTRIBUTING.md

```diff
@@ -0,0 +1,30 @@
+# Contributing to lolex
+
+## Contributing to the code base
+
+### Running tests
+
+Lolex has a comprehensive test suite. If you're thinking of contributing bug
+fixes or suggesting new features, you need to make sure you have not broken any
+tests. You are also expected to add tests for any new behavior.
+
+#### On node:
+
+```sh
+npm test
+```
+
+Or, if you prefer more verbose output:
+
+```
+$(npm bin)/mocha ./test/lolex-test.js
```

Comment:
`npx mocha` instead? Less *nix bound

Reply:
Good choice, we could also add new task to the `package.json` exrcuting `mocha ./test/lolex-test.js`, e.g `test-verbose`?